### PR TITLE
Internal: Remove unused code in Inline Fonts Icons experiment [ED-8139]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: includes/managers/icons.php [hunk 1]
```diff
@@ -229,28 +227,6 @@ private static function is_font_icon_inline_svg() {
 		return Plugin::$instance->experiments->is_feature_active( 'e_font_icon_svg' );
 	}
 
-	/**
-	 * render_svg_symbols
-	 *
-	 */
-	public static function render_svg_symbols() {
```
Comment (matipojo) [inline includes/managers/icons.php:236]: Mark as deprecated in order to avoid fatal errors, in case of using by a 3rd party plugin

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/managers/icons.php (hunk 1 above)
Comment (StyleShit) [reply includes/managers/icons.php:236]: Done

File: includes/managers/icons.php [hunk 2]
```diff
@@ -229,26 +228,13 @@ private static function is_font_icon_inline_svg() {
...
-		}
-
-		$svg .= '</svg>';
+		/**
+		 * @var $deprecation Deprecation
+		 */
+		$deprecation = Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation;
 
-		Utils::print_unescaped_internal_string( $svg );
+		$deprecation->deprecated_function( __METHOD__, '3.8.0' );
```
Comment (matipojo) [inline includes/managers/icons.php:237]: Please mark it only via PHPDoc

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/managers/icons.php (hunk 2 above)
Comment (StyleShit) [reply includes/managers/icons.php]: Done

**Pull request merged**